core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiServiceSpec.scala

```diff
@@ -50,14 +51,14 @@ class SubmissionApiServiceSpec extends ApiServiceSpec {
     }
   }
 
-  def withLargeSubmissionApiServices[T](testCode: TestApiService => T): T = {
+  def withLargeSubmissionApiServices[T](numSamples: Int)(testCode: TestApiService => T): T = {
     withDefaultTestDatabase { dataSource: SlickDataSource =>
```

Comment:
instead of calling `createLargeSampleSet` directly in the test below, you can replace `withDefaultTestDatabase` with `withCustomTestDatabase` and pass in your own `TestData` object containing your pathologically large sample set. see other uses of `withCustomTestDatabase` for examples